credoai/artifacts/data/comparison_data.py

```diff
@@ -68,6 +69,16 @@ def _validate_pairs(self):
                         f"pairs dataframe does not contain the required column '{c}'"
                     )
 
+            if len(available_columns) != 4:
+                raise ValidationError(
+                    f"pairs dataframe has '{len(available_columns)}' columns. I must have 4."
```

Comment:
"I must have 4" I love the personification of the dataframe 🤣 . "4 are required" or "It must have 4" is maybe more standard :P